flow_action_components/ExecuteSOQLQuery/force-app/main/default/classes/ExecuteSOQL.cls

```diff
@@ -1,4 +1,5 @@
-public with sharing class ExecuteSOQL {
```

Comment:
is it intentional that there are changes to ExecuteSOQL in this pull request? My guess is not....

Reply:
Good catch.. There shouldn't be any changes to SOQL in the PR. This is the version I am using on my local. :)

Reply:
I updated this branch from the remote and discarded my local changes, so these changes are gone now.